.github/workflows/run_ubuntu_system_tests.yml

```diff
@@ -0,0 +1,30 @@
+name: Run Ubuntu System Tests
+
+on:
+  workflow_call:
+  workflow_dispatch:
+
+jobs:
+  run-tests:
+    name: Run Tests
+    runs-on:
+    - self-hosted
+    - linux
+    - x64
+    - grpcbot-ubuntu
+
+    steps:
+    - name: Download Tests Artifact
+      uses: actions/download-artifact@v2
+      with:
+        name: ni-grpc-device-tests-linux-glibc2_27-x64
+
+    - run: ls -R
+
+    - name: Untar Test Build Files
+      run: >-
+        tar -xvzf ni-grpc-device-tests-linux-glibc2_27-x64.tar.gz
+
+    - name: Run System Tests
+      run: ./SystemTestsRunner --gtest_filter=-NiRFSA*:NiRFSG*:NiDCPower*CalSelfCalibrate*
```

Comment:
These tests don't work on Linux. We should fix them, if possible, but in a separate change.